spec/integration/canned_reply/canned_replies_spec.rb

```diff
@@ -7,6 +7,16 @@
     user
   end
 
+  let(:privileged_user) do
```

Comment:
I'd suggest you to keep `let` just for defining data and not for changing site settings, sign in, etc. You can use `before` for that.

Reply:
Sounds good for the site settings. Regarding `sign_in`, is it okay to emulate the `let` blocks that are already there which include it?